AdGoBye/Indexer.cs

```diff
@@ -338,21 +346,37 @@ public static void PatchContent(Content content)
 
         if (Settings.Options.EnableRecompression)
         {
-            using var uncompressedMs = new MemoryStream();
-            using var uncompressedWriter = new AssetsFileWriter(uncompressedMs);
+            if (estimatedUncompressedSize > (Settings.Options.RecompressionMemoryMaxMB * 1000L * 1000L) 
```

Comment:
Nitpick: These parentheses are redundant.
```suggestion
            if (estimatedUncompressedSize > Settings.Options.RecompressionMemoryMaxMB * 1000L * 1000L
```